lib/openstax/aws/build_image_command_1.rb

```diff
@@ -5,7 +5,7 @@ class BuildImageCommand1
       # A standardized way to use Packer to build images.
 
       def initialize(ami_name_base:, region:,
-                     verbose: false, very_verbose: false, debug: false,
+                     verbose: true, very_verbose: false, debug: false,
```

Comment:
Pretty sure it should keep defaulting to false as well and be overwritten by individual commands.